deployments/qat_plugin/qat_plugin.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-qat-plugin
+  labels:
+    app: intel-qat-plugin
+spec:
+  selector:
+    matchLabels:
+      app: intel-qat-plugin
+  template:
+    metadata:
+      labels:
+        app: intel-qat-plugin
+    spec:
+      containers:
+      - name: intel-qat-plugin
+        image: intel-qat-plugin:devel
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: pcidir
+          mountPath: /sys/bus/pci
+        - name: kubeletsockets
+          mountPath: /var/lib/kubelet/device-plugins
+      volumes:
+      - name: pcidir
+        hostPath:
+          path: /sys/bus/pci
```

Comment:
Is it really needed to mount `/sys` inside a container. In all our deployments (for the GPU and FPGA plugins) we assume `/sys` is available without specifying volume mounts.

Reply:
Its a read only file system inside the container whereas we are trying to write to files in sysfs for binding unbinding